### DISCUSSION TITLE
How change border-style of desklet photoframe?

### DISCUSSION THREAD

**Discussion by Dan-cer** [linuxmint/discussions#387] [Questions]
The nice desklet Digital picture frame (photoframe@cinnamon.org) has borders relatively thick. How could I change it? Is there a config file with css or js?
Thank you for your answer!

**Comment by Jopp-gh** [comment]
Maybe you found out by yourself by now. Just right-click on your desklet

**Reply by Dan-cer** [reply]
I have found to get some settings by rightclick. However, the question is how to make the border thinner. There isn't any option for it so far. Right?

**Reply by Jopp-gh** [reply]
Oops, you're right.

You can find the css settings in `/usr/share/cinnamon/ desklets/photoframe@cinnamon.org/stylesheet.css`.

I made my own photoframe desklet, to get more instances. If you can't modify photoframe@cinnamon.org, try to duplicate this desklet to:
`~/.local/share/cinnamon/desklets/` and modify your `stylesheet.css` to fit your needs.

**Reply by Dan-cer** [reply]
Thank you, I've found and edited it. However, I only could change the border-size that is of white color, but I meant the dark "passpartout" inside of the border. It's size is about 12 px or so.
How could I change that?

**Comment by Jopp-gh** [comment]
Did you restart cinnamon to see your changes?
Desklets need a restart, a refresh isn't enough.

You can restart cinnamon with the "looking glass" app - open system settings and configure a shortcut for it. 

Modify `padding: 0;` and `background-color:  transparent; ` there's nothing else to change. Ah,  the border. Set it to `none`.

**Reply by Dan-cer** [reply]
No change. I modified padding and bg-color and restarted cinnamon. Also I modified both in ~/.local... and /usr/share... 
The passpartout is still there. See screenshot:
![desklet](https://github.com/linuxmint/discussions/assets/800638/6dc6372a-b00f-4234-9ce9-fd6cc4010724)

**Reply by Jopp-gh** [reply]
Never change the defaults under `/usr/…`! that's no user space. Also,  always make backups of original data!  

Just make a dupicate,  rename the container, then change the name in `metadata.json`to your new name too. 
Open `desklet.js` and find `style_ class: 'photoframe-box', ` and cut it. 
Paste this string as comment in a new line like `// style_ class: 'photoframe-box', ` and save. Then restart cinnamon.

**Reply by Dan-cer** [reply]
That answered it. Thank you very much for your help! In reality, I was asking for a Mint user who wanted to know about in Linux Mint forum. I myself find the dark frame very nice! 
BTW: I haven't rename anything in `metadata.json` because I had no clue what you mean. I just copied the directory to ~/.local/share/cinnamon/desklets.  It works without renaming.
**Update:**
With comment the line I got the desklet without any frame or passpartout. That is the other extreme. How just get it thinner? 
I mean the dark gray border/frame around the image.

[Small side note: I had to remove the space in `style_  class:` ]

**Reply by Jopp-gh** [reply]
sorry for my typo, `style_  class` must be `style_class` . 

Yeah, I suggested a quick and easy tip, basically you remove  the link to `stylesheet.css`  

To modify the frame width you've to make several changes to this desklet.
Maybe I could opload my version of image frame, it has a bunch of new features…

**Reply by Dan-cer** [reply]
Yes, I would appreciate it. You mean upload as updated desklet, or "unofficial"?
